app/Jobs/SyncOpeningRegistrationDataJob.php

```
@@ -0,0 +1,94 @@
+        $safeRegistration = [
+            'registration' => [
+                'id' => data_get($details, 'registration.id'),
+                'number' => data_get($details, 'registration.number'),
+                'status' => data_get($details, 'registration.status'),
+                'files' => data_get($details, 'registration.files', []),
+            ],
+            'fileConfigurations' => data_get($details, 'fileConfigurations', []),
+            'fields' => data_get($details, 'fields', []),
+        ];
+
+        Opening::where('project_id', $this->projectId)->update([
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Opening::where().


(staticMethod.notFound)